projects/plugins/jetpack/tests/e2e/specs/blocks/free-blocks.test.js

```diff
@@ -110,4 +111,32 @@ test.describe.parallel( 'Free blocks', () => {
 			).toBeTruthy();
 		} );
 	} );
+
+	test( 'Tiled Gallery block', async ( { page } ) => {
+		await test.step( 'Can visit the block editor and add a Tiled Gallery block', async () => {
+			const blockId = await blockEditor.insertBlock(
+				TiledGalleryBlock.name(),
+				TiledGalleryBlock.title()
+			);
+			const block = new TiledGalleryBlock( blockId, page );
+			await block.addImages();
+
+			await blockEditor.openSettingsSidebar();
+			await block.linkToAttachment();
```

Comment:
Is this to ensure that the setting is there? Should we have an assertion for this too?

Reply:
Not specifically. I was thinking to check that the slideshow opens up on the frontend, but decided not to do that in this PR.